dvc/exceptions.py

```diff
@@ -268,6 +268,13 @@ def __init__(self, path, cause=None):
         )
 
 
+class FileOutsideRepoError(DvcException):
```

Comment:
How about 
```suggestion
class PathOutsideRepoError(DvcException):
```
since it might be a dir too.